### DISCUSSION TITLE
Galleries importing generated thumbnails - Help/Tips for improving galleries

### DISCUSSION THREAD

**Discussion by LoganWood2017** [stashapp/stash#4284] [Q&A]
Hello,

Stash has began importing generated thumbnails into the galleries created through autotagging performers/models. This has led to a lot of galleries for random people being created. For example for one model, I have almost 20 separate galleries, with one gallery actually containing the photos, and the others holding singular files named "SYNO_THUMB_M" that cannot be opened. Are these the generated thumbnails or something else perhaps? 

Thanks for any help! Here are my current gallery settings if it helps.

![Screenshot 2023-11-13 184451](https://github.com/stashapp/stash/assets/87874262/08558cf9-12ba-46af-935d-727eccabbae0)

**Comment by DogmaDragon** [comment]
Your Library paths and the path for generated content should be different. I believe it's not possible to add the generated path as part of Library path. 

Do you have an example path of one of the galleries?

**Reply by LoganWood2017** [reply]
Hey! I'm sorry for missing this. Here are a couple screenshots. one from the Synology Container Manager and one from the actual Stash file path.

![PXL_20240322_192456509](https://github.com/stashapp/stash/assets/87874262/a2cb21ad-08bc-4bc7-b87e-a5b75a2ddcb0)
![Screenshot 2024-03-22 132701](https://github.com/stashapp/stash/assets/87874262/b3ddbc4a-4862-4ff5-9222-1ab7c0a4aedb)

I am not quite sure how these are generating still! There are no actual files in the folders themselves, they are only present in StashApp. I wonder if it is some kind of scanning error? I noticed there are a few folders where most of the images don't get scanned as well resulting in incomplete/incorrect image count, but that may be unrelated.

Thanks for your help!

**Comment by feederbox826** [comment]
Those are automatically created synology files https://github.com/stashapp/stash/issues/3171
